source/libs/page-detect.ts

```diff
@@ -89,6 +89,8 @@ export const isRepoSettings = (): boolean => /^settings/.test(getRepoPath()!);
 
 export const isRepoTree = (): boolean => isRepoRoot() || /^tree\//.test(getRepoPath()!);
 
+export const isOwnRepo = (): boolean => isRepo() && select.exists('.reponav-item[href$="/settings"]');
```

Comment:
`select.exists('.reponav-item[href$="/settings"]')` is true even for organization repos that are technically not "own". Depending on the use-case here, it should either be renamed to `isRepoWithAccess` (better name please) or made more strict.

Reply:
It should detect orgs also, so `isRepoWithAccess` is a good alternative name.